test/conformance/event/urEventWait.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+#include <uur/fixtures.h>
+
+struct urEventWaitTest : uur::urQueueTest {
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(urQueueTest::SetUp());
+        ASSERT_SUCCESS(urMemBufferCreate(context, UR_MEM_FLAG_WRITE_ONLY, size,
+                                         nullptr, &src_buffer));
+        ASSERT_SUCCESS(urMemBufferCreate(context, UR_MEM_FLAG_READ_ONLY, size,
+                                         nullptr, &dst_buffer));
+        input.assign(count, 42);
+        ASSERT_SUCCESS(urEnqueueMemBufferWrite(queue, src_buffer, true, 0, size,
+                                               input.data(), 0, nullptr,
+                                               &event));
+        ASSERT_SUCCESS(urEventWait(1, &event));
```

Comment:
do we need to do this? the buffer write is blocking

Reply:
Fixed